src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -170,8 +175,34 @@ private static int PublishHost(ProjectContext context, string outputPath)
 
             // Copy the host
             File.Copy(hostPath, outputExe, overwrite: true);
+
+            // Publish the runtime
+            var runtimePath = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, "..", "runtime", "coreclr"));
+            if(!Directory.Exists(runtimePath))
```

Comment:
This is going to override any direct reference to the Runtime that the project has. This will prevent the project from using specific versions of the runtime.